### ISSUE TITLE
Upgrade dependencies to version 2.8.7 of RabbitMQ Java AMQP

### ISSUE THREAD

**Issue by dubiousdavid** [jduey/conduit-rabbitmq#2]
Any reason the amqp library is so far behind the current version?

**Comment by jduey** [comment]
Yeah. I've not done anything with conduit for quite awhile. On my TODO 
list is a rewrite using a protocol-arrows library that I've also not had 
time to release.

**Comment by dubiousdavid** [comment]
Thanks for the reply.  I look forward to the new release.

Lord bless,
David
